python/Ganga/GPIDev/Lib/File/LocalFile.py

```diff
@@ -224,6 +224,19 @@ def remove(self):
 
         return
 
+    def get(self, localPath=''):
```

Comment:
@milliams This has already been implemented in some form for `DiracFile` and `MassStorageFile`.
I'm aware of the remote/local issues, I've been hitting them and debugging them for a few PR now.

I'm happy to add more work to make this into a `copyToLocalPath` function (or some other name). This would effectively do what is done by providing this additional path argument to the function.
Given I've experienced friction expanding the underlying interface in the past providing an optional argument to this function seemed to be the most logical step.
It's one of these solutions or we need to expand the LocalFile schema to allow for having a LocalFile which can be moved from target to destination.

Given this allows for more flexibility to the user and I know some people use this with `DiracFile` in scripts it makes sense that this stay for existing classes).

(Yes this does imply that the GoogleFile and LCGFile need updating as a result of this PR to accept an optional input path.)